resources/Qtmodels/GeometryControls.qml

```diff
@@ -66,17 +71,38 @@ Pane {
                 anchors.verticalCenter: fileTextField.verticalCenter
                 anchors.right: parent.right
                 text: "Choose file"
-                onClicked: filePicker.open()
+                onClicked: {
+                    console.log("Opening file picker.")
```

Comment:
these should probably be removed if they are debug statements?

Reply:
Oops!